docs/i18n.md

```diff
@@ -146,17 +147,25 @@ initial bundle size. It will only be downloaded when it is needed.
 
 ### Changing the locale
 
-Because changing a locale may imply downloading new messages, a `setI18nConfiguration` method has been created
-as a helper. This method is exposed after initializing the internationalization with
-`createI18NInstance` in the [src/main.ts](../src/main.ts) file.
+There are some options in the reactive config where you can set the lang and the searchLang.
+
+- The lang is the value used for the static messages.
```

Comment:
Remove both `The` and format "lang" and "searchLang" as code.

- [ ] Remove `The` and format as code